Makefile

```diff
@@ -2,10 +2,14 @@
 # Macros #
 ##########
 
+# --- Release Version --- #
+RELEASE_VERSION = 0.3.0
```

Comment:
Version changed to 0.4.0. The version must be defined in the Makefile. It is passed as a parameter to Maven so that jars can be named accordingly. Change to Wiki will happen after the push.